imap/jmap_calendar.c

```diff
@@ -2080,7 +2080,10 @@ static void setcalendars_create(struct jmap_req *req,
     }
 
     /* Report calendar as created. */
-    *record = json_pack("{s:s}", "id", uid);
+    *record = json_pack("{s:s s:o }", "id", uid,
```

Comment:
Just so I can complain about something, there's excess whitespace after the `:o` here...

Reply:
Fixed